packages/web-components/src/components/ic-chip/ic-chip.tsx

```diff
@@ -189,6 +189,7 @@ export class Chip {
 
     return (
       <Host
+        style={{ display: visible ? "" : "none" }}
```

Comment:
I don't think we tend to add inline styles to the host of a component, would you be able to add a class for visible/not visible instead and set `display: none` in the css file?

Reply:
Following on from @GCHQ-Developer-530's comment: Doesn't setting this inline style effectively override the current  styling in the ic-chip.css on the :host element which is set to inline-block? Possibly that could be removed as part of the class based solution suggested?